SEDaily-IOS/GeneralCollectionViewController.swift

```diff
@@ -14,7 +14,7 @@ import SwifterSwift
 private let reuseIdentifier = "Cell"
 
 class GeneralCollectionViewController: UICollectionViewController {
-    let activityView = UIActivityIndicatorView(activityIndicatorStyle: .gray)
+    var skeletonCollectionView = SkeletonCollectionView(frame: .zero)
```

Comment:
Consider making member variables of a class private so they aren't exposed for use outside of the class. Even better if you can make them const using 'let' when possible too.